CorsixTH/Lua/world.lua

```diff
@@ -612,14 +612,14 @@ function World:tickEarthquake()
   else
     -- Multiplier for how much the screen moves around during the quake.
     local multi = 4
-    if (self.day > self.earthquake_stop_day) or (self.day < math.round(self.earthquake_size / 3)) then
+    if (self.day > self.next_earthquake.stop_day) or (self.day < math.round(self.next_earthquake.size / 3)) then
       -- if we are in the first two days of the earthquake, make it smaller
-      self.randomX = math.random(-(self.earthquake_size/2)*multi, (self.earthquake_size/2)*multi)
-      self.randomY = math.random(-(self.earthquake_size/2)*multi, (self.earthquake_size/2)*multi)
+      self.randomX = math.random(-(self.next_earthquake.size / 2) * multi, (self.next_earthquake.size / 2) * multi)
```

Comment:
This commit just moves variables into the next_earthquake structure. A commit further in the series replaces this whole block.